docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-REGEXFIND.xml

```diff
@@ -66,22 +66,20 @@
   find matches. The <emphasis>regex</emphasis> must be a standard Perl regular
   expression<indexterm>
       <primary>Perl regular expression</primary>
-    </indexterm>. We use third-party libraries to support this, so for
-  non-unicode <emphasis>text</emphasis>, see the Perl-compatible Regular
-  Expressions (PCRE2) documentation at <ulink
+    </indexterm>. We use third-party libraries to support this.</para>
+
+  <para>For non-unicode <emphasis>text</emphasis>, see the Perl-compatible
+  Regular Expressions (PCRE2) documentation at <ulink
   url="https://www.pcre.org/current/doc/html/pcre2pattern.html">https://www.pcre.org/current/doc/html/pcre2pattern.html</ulink>.
-  For unicode <emphasis>text</emphasis>, see the ICU docs, the sections
-  'Regular Expression Metacharacters' and 'Regular Expression Operators' at
-  <emphasis
-  role="underline">http://userguide.icu-project.org/strings/regexp</emphasis>
-  and the links from there, in particular the section 'UnicodeSet patterns' at
-  <emphasis
-  role="underline">http://userguide.icu-project.org/strings/unicodeset</emphasis>.
-  We use version 2.6 which should support all listed features.</para>
+  </para>
+
+  <para>For unicode <emphasis>text</emphasis>, <ulink
```

Comment:
We have settled on using only one third-party library for regex support (PCRE2), no matter what data type is used.  PCRE2 always supports the syntax outlined in pcre2pattern.html, even for Unicode (UNICODE or UTF8).  When searching Unicode data, PCRE2 also supports the syntax outlined in pcre2unicode.html.

To avoid confusion, the description here might want to lead with that.

Background info: Reading what I just wrote, it may seem that the latter is a superset of the former.  That is not the case.  It all comes down to things like "what is a letter" and "what is a letter" and what character set is supported.  Example: [[:digit:]] is a pattern that defines the numeric characters 0-9 in ASCII (our STRING datatype), but Unicode contains many different sets of numeric digits.  [[:digit:]] will match that same set of 0-9 characters, even in UNICODE or UTF8, but it will not match the other numeric characters; to do that you need to use \p{Nd} instead (which will get them all, even the [[:digit:]] set).

The above applies to all three REGEX ECL functions.

Can the description be reworded?